api-report/mocha-test-setup.api.md

```diff
@@ -4,8 +4,6 @@
 
 ```ts
 
-/// <reference types="mocha" />
```

Comment:
Is this what's causing the build failure? I can't update the file manually or with a build command.